exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -411,6 +411,28 @@ func (prwe *prwExporter) execute(ctx context.Context, buf *buffer) error {
 			resp.Body.Close()
 		}()
 
+		// Per the Prometheus remote write 2.0 specification, the response should contain X-Prometheus-Remote-Write-Samples-Written header.
+		// If the header is missing, it suggests that the endpoint does not support RW2 or the implementation is not compliant with the specification.
+		// Reference: https://prometheus.io/docs/specs/prw/remote_write_spec_2_0/#required-written-response-headers
```

Comment:
I have a personal preference to wrap comments at 80 or 100 columns so they are easier to read in source windows, but I won't insist on it :)